pkgs/applications/science/electronics/horizon-eda/default.nix

```diff
@@ -0,0 +1,35 @@
+{ stdenv, fetchFromGitHub, pkgconfig, sqlite, libyamlcpp, libuuid, gnome3, epoxy, librsvg, zeromq, cppzmq, glm, libgit2, curl, boost, python3, opencascade, libzip, podofo, wrapGAppsHook }:
+
+stdenv.mkDerivation rec {
+	pname = "horizon-eda";
+	version = "1.0.0";
+
+	src = fetchFromGitHub {
+		owner = "horizon-eda";
+		repo = "horizon";
+		rev = "3f777bd2501b953981d64cf2054c265ce8d65c02";
+		sha256 = "1d40lfgylwl3srqazyjgw9l3n7i82xrvkjcxvpx7ddyzy1d2aqmr";
+	};
+
+	buildInputs = [ sqlite libyamlcpp libuuid gnome3.gtkmm epoxy librsvg zeromq cppzmq glm libgit2 curl boost python3 libzip podofo opencascade ];
```

Comment:
I would prefer if you were to sort this alphabetically (easier for me to do a quick comparison against https://docs.horizon-eda.org/en/latest/build-linux.html), but more importantly place each entry on a new line, like so:
```
buildInputs = [
  boost
  curl
  ...
  zeromq
];
```